sim/src/test/scala/midasexamples/TutorialSuite.scala

```diff
@@ -182,6 +197,18 @@ class NestedModelsF1Test extends TutorialSuite("NestedModels")
 
 class MultiRegF1Test extends TutorialSuite("MultiReg")
 
+class PassthroughModelTest extends TutorialSuite("PassthroughModel") {
+  expectedFMR(2.0)
+}
+
+class PassthroughModelNestedTest extends TutorialSuite("PassthroughModelNested") {
+  expectedFMR(2.0)
```

Comment:
This is very handy. I see that it goes up to 5.0 if the passthroughs are left in.